packages/terra-clinical-data-grid/src/terra-dev-site/doc/clinical-data-grid/DatagridExamples.2/Datagrid.jsx

```diff
@@ -74,7 +77,7 @@ class StaticDataGrid extends React.Component {
 
   render() {
     return (
-      <div style={{ height: '800px' }}>
+      <div className={cx('data-grid-div-style')}>
```

Comment:
We should follow a similar approach to naming the example classes as we would for released components. I think we should avoid using the associate tag (`div`) and redundant (`style`) within the name. This class for example could be `data-grid-example` or something that describes what this example component is.

Reply:
Classnames have been renamed with component names for better understanding.